src/app/app.module.ts

```diff
@@ -4,18 +4,24 @@ import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
+import { HttpClientModule } from '@angular/common/http';
 
 import { AuthenticationModule } from './authentication/authentication.module';
+import { SurveyModule } from './survey/survey.module';
 
 @NgModule({
   declarations: [
     AppComponent
   ],
   imports: [
     BrowserModule,
-    AppRoutingModule,
     NgbModule,
-    AuthenticationModule
+    HttpClientModule,
+
+    AuthenticationModule,
+    SurveyModule,
+
+    AppRoutingModule, // keep app routing module latest to put the default and wildcard routes last
```

Comment:
Prefer putting comment above the line it refers to as it's easier to read:

```suggestion
    // Keep app routing module latest to put the default and wildcard routes last
    AppRoutingModule, 
```